frontend/amundsen_application/static/js/components/Lineage/Graph/chart.ts

```diff
@@ -327,7 +335,7 @@ export const buildNodes = (g, targetNode, nodes, onClick) => {
     .append('text')
     .attr('dy', NODE_LABEL_Y_OFFSET)
     .attr('text-anchor', 'middle')
-    .text((d, idx) => (idx !== 0 && d.data.data.name ? d.data.data.name : ''));
+    .text((d, idx) => getNodeLabel(d, idx));
```

Comment:
Nit, you don't need to pass again the arguments, you can do:
```suggestion
    .text(getNodeLabel);
```